components/infobox/commons/infobox_person.lua

```diff
@@ -119,6 +119,7 @@ function Person:createInfobox()
 			birthdate = args.birth_date,
 			birthlocation = args.birth_location,
 			deathdate = args.death_date,
+			deathlocation = args.death_location,
```

Comment:
I personally don't like adding this at all, it will for one be rarely known, for the other it is a weird thing to display.

Reply:
How detailed do you want to add these?